Tests/SwiftFormatTests/Rules/AlwaysUseLiteralForEmptyCollectionInitTests.swift

```diff
@@ -0,0 +1,117 @@
+import _SwiftFormatTestSupport
+
+@_spi(Rules) import SwiftFormat
+
+final class AlwaysUseLiteralForEmptyCollectionInitTests: LintOrFormatRuleTestCase {
```

Comment:
Could you add a couple examples in the tests using parameterized initializers (e.g., `[Int](repeating: 0, count: 10)` for arrays) just to show that they are preserved and prevent future regressions?

Reply:
Sure!